packages/vc-verification/src/resolver/issuer-resolver.ts

```diff
@@ -21,9 +22,11 @@ export interface IssuerResolver {
  */
 export class EthersProviderIssuerResolver implements IssuerResolver {
   private _domainReader: DomainReader;
+  private _dataAgregator: EVMDataAggregator;
 
-  constructor(domainReader: DomainReader) {
+  constructor(domainReader: DomainReader, dataAggregator: EVMDataAggregator) {
```

Comment:
Why is `EVMDataAggregator` being passed via the constructor? Is this because it could be shared between classes?

Reply:
Yes, so the same cached `roleDefinition` could be used to return both `issuer` and `revoker`.

Reply:
The constructor should accept the interface, not the implementation.